_data/pricing-table.yml

```diff
@@ -0,0 +1,125 @@
+              #ff4f47 98.56%
+            );
+            margin-left: 12px;
+            color:white;">Beta</label>'
+  tooltip: 
+  rows:
+    - row: 'CLI-driven updates'
+      tooltip: Use the patcher CLI to automatically update your Terraform and Terragrunt dependencies.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Patch breaking changes'
+      tooltip: When a new release contains breaking changes, Patcher can automatically apply patches from the maintainer to transform your code to work around the backward compatibility.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Automated (PR-driven) updates'
+      tooltip: When new releases come out, get automated pull-requests to update you to the new version.
+      standard: 'no'
+      enterprise: 'yes'
+
+- title: IaC Library
+  tooltip: 
+  rows:
+    - row: 'Orchestration: ECS, EKS, EC2, ASG'
+      tooltip: Reusable, battle-tested Terraform modules for deploying application workloads on top of ECS, EKS, EC2, and ASGs.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Data stores: RDS, ElastiCache, S3'
+      tooltip: Reusable, battle-tested Terraform modules for deploying data stores such as RDS, ElastiCache, and S3.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Analytics: Elasticsearch, Kafka'
+      tooltip: Reusable, battle-tested Terraform modules for deploying analytics, search, and streaming tools such as Elasticsearch, Kafka, and Kinesis.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Serverless: Lambda, API Gateway'
+      tooltip: Reusable, battle-tested Terraform modules for deploying serverless workloads on top of Lambda and API Gateway.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Self-hosting (repo-copier)'
+      tooltip: Use the repo-copier CLI tool to copy all the Terraform code to your own version control system, including updating all internal cross-references and authentication details.
+      standard: 'no'
+      enterprise: 'yes'
+
+
+- title: Support
+  tooltip: 
+  rows:
+    - row: 'Community support'
+      tooltip: Get support from the Gruntwork team and other Gruntwork customers via the <a href="https://github.com/orgs/gruntwork-io/discussions">Gruntwork Knowledge Base</a>.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Enterprise support'
+      tooltip: Get support via email, a private shared Slack channel, and scheduled video calls directly from the Gruntwork team. Includes SLAs on response times.
+      standard: 'no'
+      enterprise: 'yes'
+    - row: 'Guided onboarding'
+      tooltip: Get hands-on help, onboarding, and guidance from the Gruntwork team in setting up your DevOps Foundations.
+      standard: 'no'
+      enterprise: 'yes'
+    - row: 'Consulting'
+      tooltip: Gruntwork works directly with your team to build world-class infrastructure, which we then support & maintain on an ongoing basis.
+      standard: 'yes'
+      enterprise: 'yes'
```

Comment:
Consulting should be a red "X" for standard and the text "available" for enterprise.